TableCell/story.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import { checkA11y } from 'storybook-addon-a11y';
+import TableCell from './index';
+
+storiesOf('TableCell')
+  .addDecorator(checkA11y)
+  .add('default', () => (
+    <TableCell>
+      <div />
```

Comment:
This triggers an error since the `div` element doesn't take a `hovered` prop. Instead it would be better to pass in a small component that takes the hovered prop. [Here is an example from `Buffer Components`](https://github.com/bufferapp/buffer-components/blob/master/Button/test.jsx#L37-L38)